doc/8.topics/expr.htm

```diff
@@ -158,12 +158,12 @@
 
 <h2>
 <br>
-<font face="Helvetica, Arial, sans-serif">Expr family objects by</font>
-<font face="Helvetica, Arial, sans-serif"><a href="https://urldefense.com/v3/__http://yadegari.org/__;!!Mih3wA!FhpY8zO40hj1jgqk-RpOxiF4WYSe9sCGpdJbW5jn5NzEjaBfYg_J-MJaQnG27KlC1wWEpKjj9ShcLKPLYmo$"> Shahrokh  Yadegari</a>	</font></h2>
+<font face="Helvetica, Arial, sans-serif">Expr family of objects
+by <a href="https://yadegari.org">Shahrokh Yadegari</a></font></h2>
 
-<p>To see a directory listing of downloadable files, which also includes older releases, <a href="https://urldefense.com/v3/__http://yadegari.org/download__;!!Mih3wA!FhpY8zO40hj1jgqk-RpOxiF4WYSe9sCGpdJbW5jn5NzEjaBfYg_J-MJaQnG27KlC1wWEpKjj9Shch2SdlSY$"> click here</a>.<br>
```

Comment:
what happened to https://yadegari.org/download/ ?

Reply:
My reasoning is in https://github.com/pure-data/pddp/issues/249 and as you can see following the link, it's incomplete! We don't have any version above 0.4 (or should it be 0.40?), and it's just for Linux (not to mention an old MAX version)!!! We're now at 0.58 by the way, and we have new architectures like the apple silicon ones... raspberry pis...

Anyway, I'm also asking, just get older versions when you download older versions of Pd? Or maybe just compile old versions yourself? Also, we get lot's of downloads unrelated to expr.

Well, I did write to Shahrokh to call his attention on this issue I opened and the suggestion I did. In fact, there are some things that only he can do and change online, so he needs to be involved anyway. If he thinks we should still have a link for downloads, I can put it back.